control-plane/csi-driver/src/context.rs

```diff
@@ -70,6 +70,8 @@ pub enum Parameters {
     PvcNamespace,
     #[strum(serialize = "stsAffinityGroup")]
     StsAffinityGroup,
+    #[strum(serialize = "changefsid")]
```

Comment:
```suggestion
    #[strum(serialize = "changeFsId")]
```

Reply:
I wonder if this is clear enough?
Shouldn't it be more like `changeFsIdToPvc` or something like that?

Reply:
`changeCloneFsId` rather?

Reply:
Ah we only do it for clones right, but stll need to clarify that it changes it to the Pvc uuid